src/_includes/zebra/character-count.html

```diff
@@ -0,0 +1,54 @@
+{% comment %}
+  Character count
+
+  **Note:** This component includes changes specifically for zebra testing that diverge from the default component.
+
+  Variables:
+    - required: Boolean to determine if input is required.
+    - input_name: String of the input's name. Used to create the label and hint ID.
+    - label_text: String for the input label.
+    - character_limit: Number for the component character limit count.
+    - hint_text: String for optional input hint text.
+
+  Used in `src/_includes/zebra/forms/book-tour.html`.
+{% endcomment %}
+
+{% assign input_id = input_name | default: "character-count" %}
+{% assign input_label_id = input_id | append: "-label" %}
+
+<div class="usa-character-count">
+  <div class="usa-form-group {% if error_message  %}usa-form-group--error{% endif %}">
+    <label class="usa-label" for="{{ input_id }}">
+      {{ label_text }}
+
+      {% if required %}
+      {% include "required-hint.html" %}
+      {% endif %}
+
+      <svg
+        class="usa-icon usa-tooltip"
```

Comment:
The tooltip that appears when someone has entered an incorrect date does not necessarily tell the user that there is an error. Wondering if the word "error" needs to be put in the error suggestion or the error message somewhere. Something like, "Error: please format date as mm/dd/yyyy". Thoughts? @jaclinec @amycole501